src/ensembl/src/content/app/custom-download/CustomDownload.tsx

```diff
@@ -27,8 +50,26 @@ const CustomDownload: FunctionComponent<CustomDownloadProps> = (
   );
 };
 
-const mapStateToProps = (state: RootState): StateProps => ({
-  showPreFiltersPanel: getShowPreFilterPanel(state)
-});
+const mapStateToProps = (state: RootState): StateProps => {
+  const activeGenomeId = getCustomDownloadActiveGenomeId(state);
+
+  let showPreFiltersPanel = false;
+  if (activeGenomeId) {
+    showPreFiltersPanel = getShowPreFilterPanel(state);
```

Comment:
Maybe better call it shouldShowPreFilterPanel (otherwise it reads as if I am getting a ShowPreFilterPanel). Also, won’t it be better to check for active genome id in the selector itself?

Reply:
That's done